config/501-policy-webhook-configurations.yaml

```diff
@@ -36,16 +36,10 @@ webhooks:
       - v1
     clientConfig:
       service:
-        name: policy-webhook
+        name: webhook
         namespace: cosign-system
     failurePolicy: Fail
     matchPolicy: Equivalent
     name: validating.clusterimagepolicy.sigstore.dev
     sideEffects: None
 ---
-apiVersion: v1
-kind: Secret
-metadata:
-  name: policy-webhook-certs
-  namespace: cosign-system
-# The data is populated at install time.
```

Comment:
this is not needed anymore?

Reply:
Yes, we are getting rid and unifying all the needs to one single pod.